fixtures/cli_input/src/other_dir/mod.rs

```diff
@@ -0,0 +1,12 @@
+#[test]
+fn other_test() {
+    let mut n = 0;
+    n += 1;
+    other_noop();
+}
+
+#[allow(dead_code)]
+fn other_noop() {}
+
+// Add a reference to the adjacent_file module as requested by the reviewer
+mod other_file; 
```

Comment:
```suggestion
mod other_file; 
mod some_file; 
```
Sorry, I'm being pedantic now. But for symmetry, let's please move the tests from this file into `some_file.rs`.